tensorflow-framework/src/main/java/org/tensorflow/framework/activations/Activation.java

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.framework.activations;
+
+import org.tensorflow.Operand;
+import org.tensorflow.op.Ops;
+import org.tensorflow.types.family.TNumber;
+
+/**
+ * Abstract base class for Activations
+ *
+ * <p><b>Note:</b> The {@link #tf} attribute must be set prior to invoking the call method. See
+ * {@link #setTF(Ops)} and the constructor {@link #Activation(Ops)}.
+ *
+ * @param <T> the data type of the activation
+ */
+public abstract class Activation<T extends TNumber> {
+
+  /** The TensorFlow Ops */
+  protected Ops tf;
+
+  /**
+   * Creates the abstract class for an Activation
+   *
+   * @param tf the TensorFlow Ops
+   */
+  protected Activation(Ops tf) {
```

Comment:
We've already discussed about adding the `@Nullable` annotation but this is not part of the standard JDK and we need to bring additional dependencies, which we try to avoid as much as possible to avoid conflict with other version of that same dependency that might have been imported by the user. 

That's being said, when we discussed about it, it was related to the Core API. I'm OK though to bring more dependencies in the framework if we need to.

Reply:
The present `@Nullable` is in a private Java api and will break after Java 8.

Reply:
Ok I suggest we keep the `@Nullable` introduction for the future, outside this PR